app/controllers/medications_controller.rb

```diff
@@ -90,15 +97,26 @@ def destroy
 
   # Save refill date to Google calendar
   def save_refill_to_google_calendar(medication)
-    return unless current_user.google_oauth2_enabled? &&
-                  medication.add_to_google_cal == '1'
+    return true unless current_user.google_oauth2_enabled? &&
+                       medication.add_to_google_cal == '1'
```

Comment:
thanks for your patience and input! I need to learn to trust the instruments, instead of checking back for reviews all the time,